<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>out</class>
 <widget class="QMainWindow" name="out">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="7" column="2">
      <widget class="QPushButton" name="takeBtn">
       <property name="text">
        <string>取车</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;苹方 特粗&quot;;</string>
       </property>
       <property name="text">
        <string>欢迎取车！</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="plate"/>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>出场时间</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="returnBtn">
       <property name="text">
        <string>返回主界面</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>您的车牌号</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
